extensions/rendering-hints/README.md

```diff
@@ -0,0 +1,63 @@
+# Rendering Hints Extension Specification
+
+- **Title: Rendering Hints**
+- **Identifier: rendering_hints**
+- **Field Name Prefix: render**
+- **Scope: Item**
+- **Extension [Maturity Classification](../README.md#extension-maturity): Proposal**
+
+One of the things many people want to do with data cataloged in STAC is to render it, often doing so directly
+from the assets (usually [Cloud Optimized GeoTIFF's](http://cogeo.org)) with dynamic web tile services. To 
+perform well these services benefit from a few 'hints' that can help the renderer perform more effectively. 
+
+This document explains the fields of the STAC Rendering Hints (`render`) Extension to a STAC Item. They are meant
+to be 'hints' - things that could be discovered by inspecting the data, but provided in STAC to help out tools
+that can take advantage of them. It is written with geospatial raster data in mind, but could apply to other data types.
+
+The field names defined herein should be added as fields in the Item Properties object. When specified on an Item, 
+the values are assumed to apply to all non-thumbnail Assets in that Item. 
```

Comment:
> the values are assumed to apply to all non-thumbnail Assets in that Item

I think this is ok, but for e.g. Sentinel 2 with varied-resolution bands, some bands will have a lower `max_zoom`

Reply:
The new asset fields should cover that - you can apply the max_zoom to the asset if you want more granularity.